coriolis/osmorphing/redhat.py

```diff
@@ -188,8 +187,13 @@ def _run_dracut(self):
         self._run_dracut_base('kernel')
 
     def _run_dracut_base(self, rpm_base_name):
-        package_names = self._exec_cmd_chroot(
-            'rpm -q %s' % rpm_base_name).decode().splitlines()
+        package_names = []
+        try:
+            package_names = self._exec_cmd_chroot(
+                'rpm -q %s' % rpm_base_name).decode().splitlines()
+        except exception.CoriolisException as ex:
```

Comment:
Is there any reason why we're only catching `CoriolisException`s here?
I'm pretty sure it's not even raised by `_exec_cmd_chroot` at all.
I'd catch anything inheriting from base `Exception`